webp_backport/src/main/java/de/marcreichelt/webp_backport/FileSignatureChecker.java

```diff
@@ -0,0 +1,59 @@
+package de.marcreichelt.webp_backport;
+
+class FileSignatureChecker {
+
+    /**
+     * Check if a given byte array has the necessary WebP signature in the beginning of its content
+     *
+     * @param imgByteArr the byte array to check
+     * @return true if the byte array has the WebP signature else false
+     */
+    static boolean checkForWebP(byte[] imgByteArr) {
```

Comment:
Should be public. Also, why the name 'imgByteArr'? Why not go with a clear name 'imageByteArray'? Or even better: just 'image'? We already know it's a byte array because every IDE shows its type. :)